data/json/itemgroups/SUS/fridges.json

```diff
@@ -293,6 +293,46 @@
                   { "item": "colamdew", "container-item": "bottle_plastic", "count": [ 1, 6 ], "sealed": false }
                 ],
                 "prob": 35
+              },
+              {
+                "collection": [
+                  { "item": "zerocola", "charges": 1, "container-item": "bottle_plastic", "prob": 50, "sealed": false },
+                  { "item": "zerocola", "container-item": "bottle_plastic", "count": [ 1, 6 ], "sealed": false }
+                ],
+                "prob": 20
+              },
+              {
+                "collection": [
+                  { "item": "zeroorange", "charges": 1, "container-item": "bottle_plastic", "prob": 50, "sealed": false },
+                  { "item": "zeroorange", "container-item": "bottle_plastic", "count": [ 1, 6 ], "sealed": false }
+                ],
+                "prob": 20
+              },
+              {
+                "collection": [
+                  { "item": "zerotropical", "charges": 1, "container-item": "bottle_plastic", "prob": 50, "sealed": false },
```

Comment:
Plastic bottles hold 2 charges don't they?  Canned drinks have a single charge?

Reply:
whoops, will audit.

Reply:
> Plastic bottles hold 2 charges don't they?  Canned drinks have a single charge?

Looking at the other entries in the same bracket it seems that the whole section (I lack the correct terminology) is for unsealed bottles that has been partially consumed. All of the original entries also have a `charges: 1` stuck onto them.